tools/quickstart.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
I'm kinda opposed to checking this script in - it's *very* environment-specific, may not make sense for all devs, but has a name which implies "you should run me first!"

I think this sorta thing makes sense for developers to have / use, but it's a bit of a landmine for someone to run without a lot of context about their environment

Reply:
I think an alternative would be to include the script or substantial portions of it in the documentation, such as `how-to-run.adoc`.